### DISCUSSION TITLE
Hostile ships are missing background walls

### DISCUSSION THREAD

**Discussion by Yakunan** [FakeFishGames/Barotrauma#14953] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![Screenshot 2024-10-25 182202](https://github.com/user-attachments/assets/0a73c915-bb8b-43e2-bd1e-07156a19b01e)

Played a mission with a hostile enemy submarine.

### Reproduction steps

Play a game with a hostile submarine.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! ~~I unfortunately could not find a way to reproduce the issue. Are you using any mods by any chance? Do you remember which specific mission this was (I'm guessing "Coalition Vessel - Seek out the Coalition vessel that has been terrorizing this area and eliminate its crew")?~~

![image](https://github.com/user-attachments/assets/e3e196d0-17ed-49bb-9abd-d3e1dbb5b6c0)


EDIT: nvm, got this to happen in the multiplayer mission mode now.

**Comment by Yakunan** [comment]
I'll make sure to add more details next time. Thenks.

**Comment by Jasontti** [comment]
Tested a fix for this.
Closing discussion as resolved.
